.github/workflows/pull-request-integration.yml

```diff
@@ -44,7 +44,11 @@ jobs:
           - domain_records_info
           - domains_info
           - droplet
-          - droplet_action
+          - droplet_action_power_off
```

Comment:
The way we use `pull_request_target`, it is expected that the `droplet_action` integration test will fail, because it exists on main but no longer in this branch. This means we can't test integration automatically in the PR, but saving myself a note to run it locally to verify good functionality before ✅